lib/compilers/dotnet.ts

```diff
@@ -163,7 +163,7 @@ class DotNetCompiler extends BaseCompiler {
                 .ver 1:0:0:0
             }
             .module CompilerExplorer.dll
-            #include "${path.join(programDir, sourceFile)}"
+            #include "${process.platform === 'win32' ? path.join(programDir, sourceFile) : sourceFile}"
```

Comment:
Specially cased Windows as developers may run godbolt on Windows and I don't want to break their workflow.